src/apps/logging/logging.cpp

```diff
@@ -196,21 +199,143 @@ namespace ccfapp
       };
       // SNIPPET_END: log_record_prefix_cert
 
-      auto log_record_anonymous =
-        [this](Store::Tx& tx, nlohmann::json&& params) {
-          const auto in = params.get<LoggingRecord::In>();
-
-          if (in.msg.empty())
-          {
-            return make_error(
-              HTTP_STATUS_BAD_REQUEST, "Cannot record an empty log message");
-          }
-
-          const auto log_line = fmt::format("Anonymous: {}", in.msg);
-          auto view = tx.get_view(records);
-          view->put(in.id, log_line);
-          return make_success(true);
-        };
+      auto log_record_anonymous = [this](
+                                    RequestArgs& args,
+                                    nlohmann::json&& params) {
+        const auto in = params.get<LoggingRecord::In>();
+
+        std::vector<uint8_t> raw_quote = tls::raw_from_b64(
```

Comment:
Obviously, all of this code will go before I merge this. This is a quote I self-generated on another node and extracted the code id from, just to prove myself that it works.

Reply:
* In practice, the quote would be passed as as argument to the RPC.